release.sh

```diff
@@ -34,46 +34,58 @@ done
 
 shift $(( ${OPTIND} - 1 ))
 
-if [ $# -ne 0 ]
-then
+if [[ $# -ne 0 ]]; then
    echo "There are unexpected parameters!" 1>&2
    usage
    exit 1
 fi
 
-if [ -z "${VERSION}" ]
-then
+if [[ -z "${VERSION}" ]]; then
    echo "No version given!"
    usage
    exit 1
 fi
 
-if [ -z "${GPG_PASS}" ]
-then
+if ! [[ ${VERSION} =~ ^[0-9\.]+$ ]]; then
+    echo "Version has to match regex ^[0-9\.]+$"
```

Comment:
This will prevent RC from releasing

Reply:
and also it matches things like `.`, `000`, `5.` etc...

Reply:
fixed